src/packagedcode/models.py

```diff
@@ -135,153 +210,121 @@ class Package(object):
     DEPENDENCY_GROUPS = (dep_runtime, dep_dev, dep_optional, dep_test,
                          dep_build, dep_ci, dep_bundled)
 
+    # path to a file or directory
+    location = None
+
+    # the id of the package
+    id = StringType()
+    versioning = ModelType(Version)
+    name = StringType(required=True)
+    # this is a "short" description.
```

Comment:
Is there a way in schematics to add doc strings to the field directly rather than using comments?